main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+func calculateSum(number int) int {
+	sum := 0
+	for i := 1; i <= number; i++ {
+		sum += i
+	}
+	return sum
+}
+func hello(w http.ResponseWriter, req *http.Request) {
+	myURL := "https://localhost:8090/?n=10"
+	parsedURL, err := url.Parse(myURL)
+	if err != nil {
+		fmt.Println("Can't parse URL", err)
+		return
+	}
```

Comment:
This will never fail. Also, this parses a constant string. You should be using the URL from the request body. It should already be in a parsed state.

Reply:
How can we make the value of  n in url as variable value.
and how can we also parse more than two input value in raw query ?

Reply:
Please take a look at the request type: https://pkg.go.dev/net/http#Request
The incoming request is already parsed, so you can fetch a query value from there.

Secondly, the request Body can also contain data.